notebooks/01_data_preprocessing.ipynb

```diff
@@ -22,7 +22,8 @@
     "import scipy as sp\n",
```

Comment:
<p>I'd suggest changing the title to add RDA ... For example "Generate annual/yearly zarr stores from hourly ERA-5 netcdf files on RDA ..." </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NCAR/ERA5-dashboard-cookbook/pull/11/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Done.